apps/meteor/app/lib/server/functions/notifications/desktop.ts

```diff
@@ -22,21 +22,27 @@ export async function notifyDesktopUser({
 	room,
 	duration,
 	notificationMessage,
+	reaction,
+	reactionWithTranslation,
 }: {
 	userId: string;
 	user: AtLeast<IUser, '_id' | 'name' | 'username'>;
 	message: IMessage | Pick<IMessage, 'u'>;
 	room: IRoom;
 	duration?: number;
 	notificationMessage: string;
+	reaction: string;
+	reactionWithTranslation: string;
 }): Promise<void> {
 	const { title, text, name } = await roomCoordinator
 		.getRoomDirectives(room.t)
-		.getNotificationDetails(room, user, notificationMessage, userId);
+		.getNotificationDetails(room, user, notificationMessage, userId, reaction);
 
 	const payload = {
 		title: title || '',
 		text,
+		reactionWithTranslation,
+		reacted: reaction !== '',
```

Comment:
The reactionWithTranslation field is introduced but does not seem to have a default value or fallback mechanism when the translation fails. Ensure that reactionWithTranslation gracefully defaults to the untranslated reaction string to prevent any undefined behavior or empty notifications.

Reply:
Hi @Parvezkhan0 ,Sure I am working on it